extensions/data-transfer/portability-data-transfer-twitter/src/main/java/org/dataportabilityproject.transfer.twitter/TwitterApiWrapper.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2018 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dataportabilityproject.transfer.twitter;
+
+import org.dataportabilityproject.types.transfer.auth.AppCredentials;
+import org.dataportabilityproject.types.transfer.auth.TokenSecretAuthData;
+import twitter4j.Twitter;
+import twitter4j.TwitterFactory;
+import twitter4j.conf.ConfigurationBuilder;
+
+final class TwitterApiWrapper {
+  static  Twitter getInstance(
+      AppCredentials appCredentials,
+      TokenSecretAuthData authData) {
+
+      ConfigurationBuilder cb = new ConfigurationBuilder();
+      cb.setDebugEnabled(false)
+          .setOAuthConsumerKey(appCredentials.getKey())
+          .setOAuthConsumerSecret(appCredentials.getSecret())
+          // TODO: I think the token/secret expire, we need to check into refreshing them
```

Comment:
For refresh, there's no interface or class but for Google see data-transfer-project/extensions/data-transfer/portability-data-transfer-google/src/main/java/org/dataportabilityproject/datatransfer/google/common/GoogleCredentialFactory.java